### ISSUE TITLE
Widths for sm, md, lg do not match design spec

### ISSUE THREAD

**Issue by erikalanghauser** [AlaskaAirlines/auro-interruption#47]
## Describe the bug
Sm, Md, Lg sizes for dialog and drawer are using percentages of the screen widths without a hardcoded value for max width when screens get very large. This forces dialogs to stretch way too wide on screens 1232+. This creates a poor user experience.
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/34354487/134560373-8a2f0584-cd6c-4f6a-a226-a98eeca858d7.png)
^See the width of the flight results behind the dialog compared to width of dialog.

Please also check that the heights are coded correctly.

## To Reproduce
1. Select a Saver Fare on alaskaair.com flight results in order to see the "Get more with Main" dialog.
2. Expand your screen from 1440px, to 1680px, and beyond. 

## Expected behavior
You can see how this can be a poor experience for guests viewing on larger screens: The content is too far apart and hard to follow. It should have a hard coded max width for when screens exceed 1232px, rather than continue to stretch endlessly with the screen, in order to maintain readability and usability for guests. Please use the Figma design specs to set the hard coded values.

Large blocking modals specs:
Large modals width is fixed to **986px for viewports ≥ 1232px Desktop** 

Medium blocking modals specs:
Medium modals width is fixed to **740px for viewports ≥ 1232px Desktop**

Small blocking modals specs:
Small modals width is fixed to **740px for viewports ≥ 1232px Desktop**

This is taken from the Figma spec:  https://www.figma.com/file/Z0uBv039FxrRYVT4tnfGm9/Modal-Dialog?node-id=0%3A1699
Please also check that the heights are coded correctly according to spec.

## Screenshots
![Snip20210923_4](https://user-images.githubusercontent.com/34354487/134561235-7a2617b4-6513-4b61-a2ca-f6cdbb0e8d50.png)

**Comment by blackfalcon** [comment]
This issue has been listed for review by the team in an upcoming refinement.
